### ISSUE TITLE
[Accessibility] Hover link color in dark mode has insufficient contrast

### ISSUE THREAD

**Issue by jaimergp** [conda-incubator/conda-sphinx-theme#2]
### What happened?

This theme redefines carefully chosen defaults in the color palette to account for accessibility issues like contrast. The colors chosen are on brand with the conda ecosystem, and that's great, but there are some nuances that need to be taken into account to keep the accessibility features of the original theme.

For example, hover links color in dark mode is tricky to see:

<img width="335" alt="image" src="https://github.com/conda-incubator/conda-sphinx-theme/assets/2559438/922091be-1ee4-44f2-aa93-d4693959a9fd">

<img width="1048" alt="image" src="https://github.com/conda-incubator/conda-sphinx-theme/assets/2559438/3ed12010-842e-49de-9134-d54873f6df13">

<img width="324" alt="image" src="https://github.com/conda-incubator/conda-sphinx-theme/assets/2559438/ff1c8709-6497-47ac-8fad-9439153c4a30">

This can be further assessed  with the Chrome dev tools. Those horizontal curves in the color map are the minimum thresholds for AA and AAA; above means passing it, below means insufficient contrast.

<img width="410" alt="image" src="https://github.com/conda-incubator/conda-sphinx-theme/assets/2559438/1e769145-e418-4336-843c-b3efbb203cf6">

### Conda Info

```shell
N/A
```

### Conda Config

```shell
N/A
```

### Conda list

```shell
N/A
```

**Comment by jaimergp** [comment]
This might be useful as a CI guide: https://pydata-sphinx-theme.readthedocs.io/en/latest/community/topics/accessibility.html

**Comment by jaimergp** [comment]
Or this issue: https://github.com/pydata/pydata-sphinx-theme/issues/1052

**Comment by travishathaway** [comment]
Thanks for finding this. I don't use dark theme ever, which is why I missed it. Should be easy enough to fix.

**Comment by travishathaway** [comment]
This commit should address these issues:

- https://github.com/conda-incubator/conda-sphinx-theme/commit/257b818209d5fe23b410bbb1ab770d1efaa4e007

I've made sure to use the same colors we defined in our style guide over here:

- https://conda.org/style-guide
